Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -1096,6 +1114,7 @@ private void RenderMarkdown()
                 // Try to parse the markdown.
                 MarkdownDocument markdown = new MarkdownDocument();
                 markdown.Parse(Text);
+                Common.ImageLinkPrefix = ImageLinkPrefix;
```

Comment:
What's the purpose of having a static ImageLinkPrefix property?

Reply:
I need this set up the Link/Image prefixes [Here](https://github.com/Microsoft/UWPCommunityToolkit/pull/1639/files#diff-8482b1958b4cf87716dc7731853ff160L120) so that when links are clicked, they will have full URL. And i thought having a static property in `Common` will be the easiest way.

Reply:
My concern is that having a static property means that if I use MarkdownTextBlock in multiple places, all images must be in the same location. Wouldn't it make more sense to make this a dependency property?

Reply:
@nmetulev This is indeed a dependancy property, See [here](https://github.com/Microsoft/UWPCommunityToolkit/pull/1639/files#diff-cd726a89f8a6625dd5d4bc14c37b3675R187)

Reply:
Indeed. It just doesn't make sense to also have a static property.

Reply:
Can you point me on a way to do this?

Reply:
@nmetulev I see the problem, it's because he has to use the value [here](https://github.com/Microsoft/UWPCommunityToolkit/pull/1639/files/0fbfba755454e7574dafd99f31b79299a13c459c#diff-8482b1958b4cf87716dc7731853ff160R120) which is a helper in a different class.

@WilliamABradley does your refactoring stuff somehow make this easier?  Or any suggestions?

Reply:
It is because Processing of Blocks and Inlines has no context of the MarkdownDocument. That would require some more refactoring.

Instead it should be kept relative in the kept relative at the parser level, and then handled by the Renderer, where you can add it to the RenderContext. (This will be easier with my upcoming changes).

Reply:
Can the renderer simply add the image prefix before rendering the images? So the parser provides the absolute link (as it was written) and the renderer handles how to render it.

Reply:
@WilliamABradley I think @nmetulev's idea is better since the rendering requires full URL and we can append it at the renderer level. I will update my PR shortly.